addons/website/static/src/components/website_loader/website_loader.xml

```diff
@@ -18,14 +18,16 @@
                 <br/>pointer to build the perfect page in 7 steps.
             </t>
         </p>
-        <div t-if="featuresInstallInfo.total > 0"
-        class="o_website_loader_progress progress rounded-pill mt-3">
-            <div role="progressbar" aria-label="Installing your features"
-            class="progress-bar bg-success progress-bar-striped progress-bar-animated rounded-pill"
-            t-att-aria-valuemax="featuresInstallInfo.total"
-            t-att-aria-valuenow="featuresInstallInfo.nbInstalled"
-            t-attf-style="width: #{featuresInstallInfo.nbInstalled * 100 / featuresInstallInfo.total}%">
+        <div t-if="featuresInstallInfo.total > 0">
+            <div class="o_website_loader_progress progress rounded-pill mt-3">
+                <div role="progressbar" aria-label="Installing your features"
+                class="progress-bar bg-success progress-bar-striped progress-bar-animated rounded-pill"
+                t-att-aria-valuemax="featuresInstallInfo.total"
+                t-att-aria-valuenow="featuresInstallInfo.nbInstalled"
+                t-attf-style="width: #{featuresInstallInfo.nbInstalled * 100 / featuresInstallInfo.total}%">
+                </div>
             </div>
+            <span t-esc="featuresInstallInfo.nbInstalled * 100 / featuresInstallInfo.total"/>%
```

Comment:
Was for debugging purpose, but wouldn't it be nice to show to apps total (instead of %) ?
Like [23/45], then [33/45] etc?
I think I wrote something about it in the spec, it would give the impression than something is being done (and would be more "understandable" than % -> You know there is 45 moduels to be installed, and this is doing the install of the 23th one

Reply:
That's +/- what I initially did, but @pierre-pde didn't like it. He considers that the progressbar gives enough information in itself.

I'll look at the rest on Monday, thanks!

Reply:
If we have the progress bar working smoothly (and not going from 0 to 100 in one go at 90% of the process) it's maybe something we can live with indeed.
Even if I think having the number increase every 2 sec will be noticeable, while the progress bar moving 3% every 2 seconds is barely noticeable in itself, but as you prefer.

Reply:
I tested it in various situations, among which the single eCommerce module installation, and you do see the progress.

If I remember correctly, @pierre-pde's opinion was 1) that the modules' numbers don't evoke anything to the base user, 2) that a % is redundant with the progressbar, 3) it takes space for no real added value. So the progressbar is enough.

Reply:
> I tested it in various situations, among which the single eCommerce module installation, and you do see the progress.

Weird, because you basically had only the direct dependencies of the module in your controller.
So for `website_sale`, you only has like 5-7 modules like UOM etc.
This was because of the `auto_install_required` part of your condition.
So I don't see how it was really behaving smoothly, what am I missing? (and I was in multi workers)

Basically, this is what I experienced with your diff before I pushed my wip: https://watch.screencastify.com/v/jCsGPVxnsqt26Q31RQ4I

> 1) that the modules' numbers don't evoke anything to the base user, 2) that a % is redundant with the progressbar, 3) it takes space for no real added value. So the progressbar is enough.

Well if the progress bar is progressing constantly / smoothly, I don't mind anyway, it will be enough indeed, but just to reply (ignore it):
1.  I am sure everyone understand what X/Y means (especially with a word next to it), it's literally what you see everywhere, PlayStation updates, TV update, phone update etc.
2. Yes but only if the progress bar move constantly
3. Not really, as the progress bar is taking a whole "line" on the screen while it occupied a very minor part of it (like 5-10%?), so adding a small % next to it would not change much. Better, it would be part of the progress bar, inside it.

Reply:
About the 1st part: yes, sorry I actually meant _now_ you see the progress with the ecommerce module. I just wanted to say that the case you pointed out was resolved and that I feel the progress on the progress bar itself is noticeable with those adjustments.
(Basically answering to:)
> the progress bar moving 3% every 2 seconds is barely noticeable in itself

Reply:
It indeed now really looks great :+1: 
See https://watch.screencastify.com/v/GSOqa5LQttsNeds8rIu6 on your latest runbot.

I am a bit skeptical about the part where all modules are installed but there is still some time before the configurator is really finished, a few seconds, but it will probably be okay as 95% of the processing time will be covered by the progress bar, the impression of "this is stuck" should be gone, **let's not care about that for now**, that would be premature optimization. (Could have tried to do something if we kept the % next to the progress bar, at 100% it could have mentioned something like "Module install done, now cleaning xyz", like installation process usually do when you are at 100% but still something being processed.).